components/ui/CustomLink.tsx

```diff
@@ -27,9 +27,9 @@ export default function CustomLink({
   };
 
   const style = clsx(
-    lightColor ?? defaultStyle.lightColor,
-    `dark:${darkColor?.replace("dark:", "")}` ?? defaultStyle.darkColor,
-    `hover:${hover?.replace("hover:", "")}` ?? defaultStyle.hover
+      lightColor !== undefined ? lightColor : defaultStyle.lightColor,
+      darkColor !== undefined ? `dark:${darkColor?.replace("dark:", "")}` : defaultStyle.darkColor,
+      hover !== undefined ? `hover:${hover?.replace("hover:", "")}` : defaultStyle.hover
```

Comment:
thats good, new idea, what if when `lightColor` or `darkColor` or `hover` is set to null, even the default value isnt used either?